source/ContextualPopup.js

```diff
@@ -542,6 +542,10 @@
 			this.inherited(arguments);
 			this.alterDirection();
 			this.showHideScrim(this.showing);
+
+			if(this.spotActivatorOnClose) {
+				enyo.Spotlight.spot(this.activator);
```

Comment:
@dmikey , Sorry for the confusion.
When I see the popup showingChanged code, popup is checking that current focus is on popup before actually set focus. The popup showing property can be set in programming way (from luna call response handler or setTimeout handler), If focus can be on outside of popup when showingChange is called then we need to filter out these cases before change focus.